src/components/eip4626/User.tsx

```diff
@@ -36,10 +51,45 @@ export default function User() {
     );
   });
 
+  const userVaultSectionTabs = {
+    vaultInfo: {
+      title: "Vault info",
+      _render: () => (
```

Comment:
Lets drop those iterations and _renders in favour of just having the content in place, see the discussion in the further comments of this PR.